src/django_perf_rec/utils.py

```diff
@@ -38,9 +38,9 @@ def current_test() -> TestDetails:
     assert frame is not None
     try:
         while True:
-            details = _get_details_from_test_function(
+            details = _get_details_from_pytest_request(
                 frame
-            ) or _get_details_from_pytest_request(frame)
+            ) or _get_details_from_test_function(frame)
```

Comment:
@adamchainz this is the part that I'm not confident is the right solution.

Reply:
Prioritizing details from pytest seems sensible to me. If we implement the below "save the request" optimization, it could actually bypass stack inspection, which is pretty slow.